pkg/actions/cluster/unowned.go

```diff
@@ -78,6 +78,10 @@ func (c *UnownedCluster) Delete(waitInterval time.Duration, wait bool) error {
 		return err
 	}
 
+	if err := logUndeletedStacks(c.ctl.NewStackManager(c.cfg)); err != nil {
+		return err
+	}
+
 	logger.Success("all cluster resources were deleted")
 	return nil
```

Comment:
I don't think this is great UX. We should either:
1. Print the warning and change the wording of the success message to "deletion complete"
2. Exit non-zero to indicate to the user that we didn't delete everything. Print a suggestion of manually deleting the cloudformation stacks
3. We should just delete the stacks if we find any rather than logging. However randomly deleting stacks that might have other dependencies that also need to be cleaned up could be problematic

Reply:
I'm for 2), if there are still stacks `eksctl` "knows" belongs to it but we didn't delete them, then it's definitely a failure to delete, IMO